src/libsyntax/diagnostic.rs

```diff
@@ -550,32 +550,17 @@ impl EmitterWriter {
         // Display only the first MAX_LINES lines.
         let all_lines = lines.lines.len();
         let display_lines = cmp::min(all_lines, MAX_LINES);
-        let display_line_infos = &lines.lines[..display_lines];
         let display_line_strings = &line_strings[..display_lines];
 
-        // Calculate the widest number to format evenly and fix #11715
-        assert!(display_line_infos.len() > 0);
-        let mut max_line_num = display_line_infos[display_line_infos.len() - 1].line_index + 1;
-        let mut digits = 0;
-        while max_line_num > 0 {
-            max_line_num /= 10;
-            digits += 1;
-        }
-
+        assert!(display_line_strings.len() > 0);
```

Comment:
Since our idea is to drop displaying file paths for the latter part of the span, we'd be reverting the patches made for #11715 and some of its sub-issues. Is that okay?